.github/workflows/deploy.yml

```diff
@@ -1,4 +1,4 @@
-name: Deploy
+name: Deployment
```

Comment:
Why change this name? They are usually "Verb" or "Verb noun", like "Test" or "Deploy production".

Reply:
This is to make title consistent with other workflows - we have "Pull request" (which I don't like too much and I would recommend to rename it and "CodeQL Analysis"